.github/workflows/go-ci.yml

```diff
@@ -26,6 +26,6 @@ jobs:
           # rules: https://golangci-lint.run/usage/quick-start/
           args: -E asciicheck,goimports
       - name: Test and generate coverage
-        run: go test -coverprofile=coverage.out -covermode=atomic ./...
+        run: go test -coverpkg=all -coverprofile=coverage.out -covermode=atomic ./...
```

Comment:
I see all examples/docs use `coverpkg=./...`, but `all` was used in the original go changelog of `coverpkg`. And `go help packages` describes them somewhat differently. Shall we change it to `./...`? Kind of curious if we get different results, and it matches the package selector nicely.

Reply:
It seems like using `./...` covers a smaller subset, but also seems like what we want - apparently `all` includes stuff like even the standard library.